### PR TITLE
Add chartcontent spec

### PR DESCRIPTION
**Pull request by galal-hussein**
https://github.com/rancher/rke2/issues/34

### CODE REVIEW

File: pkg/helm/controller.go [hunk 1]
```diff
@@ -402,3 +383,80 @@ func setProxyEnv(job *batch.Job) {
...
+	}
+}
+
+func setValuesConfigMap(job *batch.Job, chart *helmv1.HelmChart) *core.ConfigMap {
+	configMap := valuesConfigMap(chart)
+	if configMap == nil {
+		return nil
+	}
+
+	job.Spec.Template.Spec.Volumes = []core.Volume{
```
Comment (dweomer) [inline pkg/helm/controller.go:412]: shouldn't this be an append operation?

File: pkg/helm/controller.go [hunk 2]
```diff
@@ -402,3 +383,80 @@ func setProxyEnv(job *batch.Job) {
...
+				ConfigMap: &core.ConfigMapVolumeSource{
+					LocalObjectReference: core.LocalObjectReference{
+						Name: configMap.Name,
+					},
+				},
+			},
+		},
+	}
+
+	job.Spec.Template.Spec.Containers[0].VolumeMounts = []core.VolumeMount{
```
Comment (dweomer) [inline pkg/helm/controller.go:425]: https://github.com/rancher/helm-controller/pull/53/files#r441868717

File: pkg/helm/controller.go [hunk 3]
```diff
@@ -402,3 +383,80 @@ func setProxyEnv(job *batch.Job) {
...
+	return configMap
+}
+
+func setContentConfigMap(job *batch.Job, chart *helmv1.HelmChart) *core.ConfigMap {
+	configMap := contentConfigMap(chart)
+	if configMap == nil {
+		return nil
+	}
+
+	job.Spec.Template.Spec.Volumes = []core.Volume{
```
Comment (dweomer) [inline pkg/helm/controller.go:441]: https://github.com/rancher/helm-controller/pull/53/files#r441868717

File: pkg/helm/controller.go [hunk 4]
```diff
@@ -402,3 +383,80 @@ func setProxyEnv(job *batch.Job) {
...
+				ConfigMap: &core.ConfigMapVolumeSource{
+					LocalObjectReference: core.LocalObjectReference{
+						Name: configMap.Name,
+					},
+				},
+			},
+		},
+	}
+
+	job.Spec.Template.Spec.Containers[0].VolumeMounts = []core.VolumeMount{
```
Comment (dweomer) [inline pkg/helm/controller.go:454]: https://github.com/rancher/helm-controller/pull/53/files#r441868717

File: pkg/helm/controller.go (hunk 2 above)
Comment (galal-hussein) [reply pkg/helm/controller.go:425]: yup thats a bug, going to fix that

**Pull request merged**